src/gui/TrackContainerView.cpp

```diff
@@ -377,7 +377,7 @@ void TrackContainerView::dropEvent( QDropEvent * _de )
 		//it->toggledInstrumentTrackButton( true );
 		_de->accept();
 	}
-	else if( type == "samplefile" || type == "pluginpresetfile" 
+	else if( type == "samplefile" || type == "pluginpresetfile"
```

Comment:
Can be reverted, but not necessary.

Reply:
I tried it, Qtcreator removes it on save. :) I can force it by using an other editor if it is necessary.